theano/tests/test_printing.py

```diff
@@ -108,11 +108,11 @@ def test_min_informative_str():
 
     mis = min_informative_str(G).replace("\t", "        ")
 
-    reference = """A. Elemwise{add,no_inplace}
- B. C
- C. Elemwise{add,no_inplace}
-  D. D
-  E. E"""
+    reference = #A. Elemwise{add,no_inplace}
+ # B. C
+ # C. Elemwise{add,no_inplace}
+ # D. D
+ # E. E
```

Comment:
undo. This isn't a comment, but a python string we need.

Reply:
sure